Makefile

```diff
@@ -356,9 +356,8 @@ generate-operator-bundle: manifests
 	# This is only needed until CRD conversion support is released in OpenShift 4.6.x/Operator Lifecycle Manager 0.16.x
 	scripts/add-openshift-cert-handling.sh
 	# Inject the container reference into the bundle.
-	scripts/inject-container-reference.sh "$(PUBLIC_REPO)@$(LATEST_TAG)"
+	scripts/inject-container-reference.sh "$(PUBLIC_REPO):$(LATEST_TAG)"
```

Comment:
Is there a markdown document describing the usage of this build target (i.e. what to do once you've run it?) If so might be worth referencing that markdown via a comment in the makefile.

Reply:
Not yet - that's the doc I mention in the description.